backend/queries/user_serach_settings_query.py

```diff
@@ -0,0 +1,74 @@
+from enums import Category, Country, Language, Source
+from models.user_search_settings import UserSearchSettings
+from schemas.user_search_settings_schema import UserSearchSettingsObject
+from sqlalchemy.orm import Session
+
+
+class UserSearchSettingsQuery:
+    def create_user_search_settings(
+        self, db: Session, settings: UserSearchSettingsObject
+    ):
+        new_settings = UserSearchSettings(
+            id=settings.id,
+            country=settings.country,
+            category=settings.category,
+            source=settings.source,
+            language=settings.language,
+            user_id=settings.user_id,
+        )
+        db.add(new_settings)
+        db.commit()
+        db.refresh(new_settings)
+        return new_settings
+
+    def get_users_settings(self, db: Session):
```

Comment:
```suggestion
    def get_users_settings(self, db: Session) -> UserSearchSettings:
```
Can you add returning type?

Reply:
and in other places also :D